vignettes/admiral.Rmd

```diff
@@ -23,7 +23,7 @@ processed dataset. This modular approach makes it easy to adjust code by adding,
 removing, or modifying derivations. Each derivation is a function call. Consider
 for example the following script which creates a (very simple) ADSL dataset.
 
-```r
+```{r, message=FALSE, warning=FALSE}
 library(dplyr)
 library(lubridate)
 library(admiral)
```

Comment:
Please add `library(cdiscpilot)` here. That's the reason why the checks fail.

Reply:
Added library(admiral.test).